test/fixtures/cookbooks/install_varnish/recipes/full_stack.rb

```diff
@@ -19,10 +19,9 @@
   variables(
     config: {
       backend_host: '127.0.0.1',
-      backend_port: '8080',
+      backend_port: '8080'
```

Comment:
Master is still running CircleCI and rubocop currently, had to remove that to get it to pass (https://circleci.com/gh/sous-chefs/varnish/709). We'll want to get that in line with the other cookbooks but will probably have to do that in another pull request, updated the link on the README though so at least that isn't broken anymore.